src/utils/messages.js

```diff
@@ -31,18 +31,15 @@ function displayMessageWithSnackbar({
     level,
     persistent,
 }) {
-    let message;
+    let message = intlRef.current.formatMessage({ id: errorMessage });
```

Comment:
Remove these modifications

Reply:
We will have more message in the scnakbar.

Reply:
It's fixed.